projects/pgai/pgai/vectorizer/generate/docker-compose.yaml

```diff
@@ -1,7 +1,10 @@
 name: pgai
 services:
   db:
-    image: timescale/timescaledb-ha:pg17
+    build:
+      context: ../../../../extension
+      dockerfile: Dockerfile
+      target: pgai-test-db
```

Comment:
💯 I was always doing this change locally but not committing 😅

Reply:
WDYT about creating a separate PR with this change targeting `main` branch so other devs can benefit from it asap.